srcpkgs/php/template

```diff
@@ -1,15 +1,15 @@
 # Template file for 'php'
 pkgname=php
 version=7.4.14
-revision=3
+revision=4
```

Comment:
Maybe add an install.msg for php-pear saying that Pear has been removed? And make `php-pear` a dumy transitional package for now.

We can't make it depend on `pear` due to cyclic deps, and `pear` can't have a `php-pear` subpackage without a huge list of reverts.